.github/workflows/build.yml

```diff
@@ -124,6 +124,7 @@ jobs:
           rust: true
           rust-wasm: true
           rust-cache: true
+          vcpkg: "${{ matrix.os == 'windows-latest' }}"
```

Comment:
we may need similar change in release.yml as well. (I have volunteered to consolidate build/release actions to use same source to be consistent with each other).